create-release/index.js

```diff
@@ -37,14 +37,16 @@ async function main() {
         const env = getEnvironment(tagSplitted[2]);
         previousTag = getPreviousTagWithEnv(allTags.data, env);
     }
+
+    let previousTagName = previousTag ? previousTag.name : "No previous tag";
```

Comment:
```suggestion
    const previousTagName = previousTag ? previousTag.name : "No previous tag";
```

By the way, is this really working? 
I can't find mention of `"No previous tag"` in the [github api doc](https://docs.github.com/en/rest/releases/releases#generate-release-notes-content-for-a-release)

Reply:
Actually, I thought it was a declarative field where I could put anything.
I should have read the doc before, it seems that if there is not previous tag, we have to let it empty. I'll change this